packages/contracts/contracts/Poll.sol

```diff
@@ -181,6 +185,9 @@ contract Poll is Clone, Params, Utilities, SnarkCommon, IPoll {
     // store message batch size
     messageBatchSize = _messageBatchSize;
 
+    // store  state tree depth
+    stateTreeDepth = _stateTreeDepth;
+
     // store tree depth
     treeDepths = _treeDepths;
```

Comment:
why not store it inside treeDepths (seems like the best place logically to keep it)?